docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -35,6 +35,7 @@
 |spring.cloud.loadbalancer.health-check.interval | 25s | Interval for rerunning the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.path |  | 
 |spring.cloud.loadbalancer.hint |  | Allows setting the value of <code>hint</code> that is passed on to the LoadBalancer request and can subsequently be used in {@link ReactiveLoadBalancer} implementations.
+|spring.cloud.loadbalancer.retry.avoidPreviousInstance.enabled | true | Instantiates AvoidPreviousInstanceRoundRobinLoadBalancer if Spring-Retry is in the classpath.
```

Comment:
Add back ticks around AvoidPreviousInstanceRoundRobinLoadBalancer

Reply:
This file is automatically generated.